pages/development/theming.mdx

```diff
@@ -1,21 +1,270 @@
 # Theming
 
-Embeddable lets you control the look and feel of your analytics without hacking at the CSS. Currently, we provide a **constants file** in the [Starter Components](/dashboards/starter-components) where you can tweak key styling properties such as:
+Embeddable theming is a straightforward approach to styling your dashboard that allows you to control a large number of values in a single TypeScript object.
 
-- **Fonts** (family, size, weight)  
-- **Chart Colors** (e.g., primary series, hover states)  
-- **Date display formats** (e.g. `month: 'MMM yy'`).
+<Callout emoji="⚠️">
+To use theming, you must upgrade to Vanilla Components v1. Full information on how to do so can be found [on our upgrading page](/development/theme-upgrading).
+</Callout>
 
-Since you have access to the underlying code, you can also make any other styling changes you want directly within the components. 
+### The Basics
 
-## Upcoming: Multiple Theme Support
+If you use Vanilla Components, you will be provided with a complete default theme under the hood (the same colors and appearance you can see on our example components page), and can then override as many or as few values as you want, within what the theme supports. This is a much larger variety of values than was previously available to change, and we’ll list them all in an appendix to this document, but if all you want to do is change a few chart colors and the font, your theme object could look like this:
```

Comment:
@cwbuecheler could "our example components page" be a link?

Reply:
@cwbuecheler "This is a much larger variety of values than was previously available to change" I would leave out sentences like this as they only make sense to existing customers (and they won't make sense to anybody in 6 months from now)

Reply:
@cwbuecheler "Vanilla components" could also link to the vanilla components repo.